hal/armv8m/nrf/91/config.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Phoenix-RTOS
+ *
+ * plo - operating system loader
+ *
+ * Platform configuration file
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _CONFIG_H_
+#define _CONFIG_H_
+
+#ifndef __ASSEMBLY__
+
+#include "nrf91.h"
+#include "peripherals.h"
+#include "../types.h"
+
+#include <phoenix/arch/syspage-nrf91.h>
+#include <phoenix/syspage.h>
+
+#include "../../cpu.h"
+
+#define PATH_KERNEL "phoenix-armv8m33-nrf9160.elf"
```

Comment:
I think that this target name should be something like nrf91xx or nrf91 - you specify HAL for nrf91 family but specific MCU is specified in other places

Reply:
As we agreed I would leave it as it is